wagtail/blocks/stream_block.py

```diff
@@ -710,7 +710,9 @@ def _prefetch_blocks(self, type_name):
         raw_values = OrderedDict(
             (i, raw_item["value"])
             for i, raw_item in enumerate(self._raw_data)
-            if raw_item["type"] == type_name and self._bound_blocks[i] is None
+            if raw_item
```

Comment:
Should I place a comment above like:
```python
# May be None if inserted with insert()
```

Reply:
Actually, I could have swapped the existing if condition from `raw_item["type"] == type_name and self._bound_blocks[i] is None` to `self._bound_blocks[i] is None and raw_item["type"] == type_name` and it should fix the issue too :sweat_smile: 
Which version should I use?

Reply:
Ok, I've now flipped the original if-condition, therefore I think no more explanation is needed.